src/VisualStudio/Core/Test/Help/HelpTests.vb

```diff
@@ -2092,6 +2092,20 @@ Class C
     End Sub
 End Class]]></a>.Value, "System.Object.GetType")
         End Sub
+
+        <WorkItem(4150, "https://github.com/dotnet/roslyn/issues/4150")>
+        <Fact, Trait(Traits.Feature, Traits.Features.F1Help)>
+        Public Sub TestPropertyFromMemberAccess()
```

Comment:
Are there tests for locals/aliases/parameters? (I'd guess not, because there was no test for properties that needed to be fixed here.) If not, this might be a good opportunity to add some coverage of this use-the-type-instead-of-the-name feature.